package-lock.json

```diff
@@ -0,0 +1,3343 @@
+{
```

Comment:
The project uses yarn, could you remove and do `yarn` and commit that lock instead?

Reply:
👍